backend/app/services/messages.js

```diff
@@ -0,0 +1,64 @@
+const request = require('request-promise');
+const config = require('../../config');
+
+module.exports = {
+    SYSTEM_MESSAGE_USER_TOKEN_FIELD: 'SYSTEM_MESSAGE_USER_TOKEN',
+    SYSTEM_MESSAGE_SUBJECT: 'System Message',
+    createSystemMessageUser: function() {
+        const path = '/user';
+
+        const requestOptions = {
+            url: config.authService + path,
+            method: 'POST',
+            json: {
+                username: config.systemMessageUser,
+                email: config.systemMessageUser,
+                password: config.systemMessagePassword,
+                scopes: [],
+            }
+        };
+
+        return request(requestOptions)
+        .catch((err) => {
+            if (err.statusCode === 400) {
```

Comment:
Do we want to handle other status codes? It seems like the mostly likely case is actually a `401` if the system user credentials end up invalid.

Reply:
If failing to authenticate the SYS_MESSAGE_USER should be fatal to the application, we can halt on these errors. I figured it shouldn't be fatal, but maybe we would rather find out at boot time that system messages are failing than at runtime.

Reply:
Agreed, some sort of valid credentials check.